packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -102,5 +103,17 @@ export const VtexCommerce = (
         )
       },
     },
+    person: (): Promise<Person> =>
```

Comment:
[⛏️  Nitpick] Just out of curiosity, why name this `person` instead of something more generic, like `session` or `profile`?